process/smartContract/process.go

```diff
@@ -301,6 +301,12 @@ func (sc *scProcessor) doExecuteSmartContractTransaction(
 		return vmOutput.ReturnCode, nil
 	}
 
+	err = sc.gasConsumedChecks(tx, vmOutput)
+	if err != nil {
+		log.Error("gasConsumedChecks with problem ", "err", err.Error(), "txHash", txHash)
+		return vmcommon.ExecutionFailed, sc.ProcessIfError(acntSnd, txHash, tx, err.Error(), []byte("gas consumed exceeded"), snapshot, vmInput.GasLocked)
```

Comment:
is this still backwards-compatible?

Reply:
yes. it is. only in asynccall we had these issues. plus this shouldn't ever happen